libr/core/cmd_search.c

```diff
@@ -916,11 +916,13 @@ static void print_rop (RCore *core, RList *hitlist, char mode, bool *json_first)
 	unsigned int size = 0;
 	RAnalOp analop = {0};
 	RAsmOp asmop;
+	Sdb *db = sdb_ns (core->sdb, "rop", false);
```

Comment:
only create the db if its suposed to be used. so move the conditionals up to here